src/dotnet-user-secret/project.json

```diff
@@ -14,18 +14,23 @@
     "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
     "Microsoft.Extensions.Logging": "1.0.0-*",
+    "Microsoft.NETCore.Platforms": "1.0.1-rc2-23720",
```

Comment:
I think we should use a * version because we update this package everytime we pick a new coreclr